recipes/date/all/conanfile.py

```diff
@@ -23,14 +24,14 @@ class DateConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "header_only": [True, False],
-        "use_system_tz_db": [True, False],
```

Comment:
Please, revert it. The option `use_system_tz_db` follows a primary option from the upstream, changing it will transform the package to another thing and will break users.

In case you want to use tz data, it should be another option, but not changing the default behavior from the upstream.

Reply:
> Feedback I've received in past recipe reviews has been that the conan package interface doesn't need to match the CMake interface exactly.

True, but there is a limit for it. This PR is changing not only an option, but adding a behavior. Please, open an issue to the upstream and discuss it first there.